_build/data/docs/commandline.rst

```diff
@@ -26,37 +53,56 @@ The EventGhost main executable accepts the following command line arguments:
     Shorter alias for the :option:`-hide` option.
 
 
-.. cmdoption:: -file <filename>
+.. cmdoption:: -multiload
 
-    Opens the XML configuration file <filename> on start-up (instead of the 
-    last loaded file). 
+    Allow multiple sessions of EventGhost on the same computer.
 
 
-.. cmdoption:: -f <filename>
+.. cmdoption:: -m
+
+    Shorter alias for the :option:`-multiload` option.
 
-    Shorter alias for the :option:`-file` option.
+
+.. cmdoption:: -netsend <host>:<port> <password> <eventname> [<payload> ...]
+
+    This one is similar to the :option:`-event` option, but sends the event
+    <eventname> through TCP/IP like the 'Network Event Sender' plugin does. It
+    will not start EventGhost, so it can be used as a little helper tool for
+    other applications or .BAT files to send events to a remote machine.
+    <host> has to be the IP or host name of the target machine. <port> and
+    <password> are the options that you have configured on the target
+    machine's 'Network Event Receiver' plugin.
 
 
 .. cmdoption:: -n <host>:<port> <password> <eventname> [<payload> ...]
 
-    This one is similar to the :option:`-event` option, but sends the event 
-    <eventname> through TCP/IP like the 'Network Event Sender' plugin does. It 
-    will not start EventGhost, so it can be used as a little helper tool for 
-    other applications or .BAT files to send events to a remote machine. 
-    <host> has to be the IP or host name of the target machine. <port> and 
-    <password> are the options that you have configured on the target 
-    machine's 'Network Event Receiver' plugin. 
+    Shorter alias for the :option:`-netsend` option.
 
 
-.. cmdoption:: -translate
+.. cmdoption:: -plugin <egplugin>
 
-    Starts EventGhost's translation editor. 
-    
+    Opens the plugin archive file <egplugin> for installation in EventGhost.
+    Plugins installed through a <egplugin> plugin archive file are stored in the
+    %AppData%\EventGhost\plugins folder and take precedence over plugins in the
+    %ProgramFiles%\EventGhost\plugins folder.
```

Comment:
I would have to check on this. But i do know there is an issue that was brought up and I made a change on and I have yet to hear back if it works properly. But it has to do with EG running elevated and then double clicking the .egplugin file and an error being created because you have to run the .egplugin file elevated as well.